src/util.js

```diff
@@ -504,6 +504,26 @@ export function isPending(threadState) {
     return PENDING_STATES.indexOf(threadState) > -1;
 }
 
+/**
+ * Checks whether an annotation thread has valid min/max boundary coordinates
+ *
+ * @param {AnnotationThread} thread Annotation thread location object
+ * @return {boolean} Whether or not the annotation has valid boundary coordinates
+ */
+export function hasValidBoundaryCoordinates(thread) {
+    return !!(
+        isHighlightAnnotation(thread.type) ||
+        (thread.minX &&
```

Comment:
AFAIK `NaN` is neither true nor false (`NaN == true` is false, and `NaN == false` is false). you may not need to do your `Math.isNaN()` checks here.

Reply:
👍